### PR TITLE
LPS-144314

### PR DESCRIPTION
**Pull request by ozysouza**
https://issues.liferay.com/browse/LPS-144314

cc @david-gutierrez-mesa 
cc @Tim-Cao

### CODE REVIEW

File: portal-web/test/functional/com/liferay/portalweb/macros/layout/pagesadmin/PagesAdmin.macro [hunk 1]
```diff
@@ -714,6 +714,12 @@ definition {
 		Click(locator1 = "MenuItem#DROPDOWN_MENU_ITEM");
 	}
 
+	macro gotoCollectionEntryEllipsisItem {
+		LexiconTable.clickEllipsisItem(
+			item = "${item}",
+			tableEntry = "${tableEntry}");
+	}
+
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/macros/layout/pagesadmin/PagesAdmin.macro:722]: Keep in the original file

File: portal-web/test/functional/com/liferay/portalweb/macros/layout/pagesadmin/PagesAdmin.macro [hunk 2]
```diff
@@ -1050,6 +1050,29 @@ definition {
 			value1 = "/${pageName}");
 	}
 
+	macro viewMenuItemNotPresent {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/macros/layout/pagesadmin/PagesAdmin.macro:1053]: Keep the orignal logic just rename to cannotDeletePagesAssociatedWithSiteTemplate

File: portal-web/test/functional/com/liferay/portalweb/macros/layout/pagesadmin/PagesAdmin.macro [hunk 3]
```diff
@@ -1211,6 +1211,14 @@ definition {
 		}
 	}
 
+	macro viewSelectCategoryNotPresent {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/macros/layout/pagesadmin/PagesAdmin.macro:1222]: viewVocabularyNotPresent

**Review by Tim-Cao — COMMENTED** [review]
Hi @ozysouza, could you update according to comments? Thanks

**Comment by ozysouza** [comment]
> Hi @ozysouza, could you update according to comments? Thanks

Hi @Tim-Cao, PR updated following your comments. Would you have a look at? Thanks!

**Comment by ozysouza** [comment]
ci:test:sf

**Comment by Tim-Cao** [comment]
LGTM!

**Comment by Tim-Cao** [comment]
ci:forward

**Pull request closed without merging**